videoframes.py

```diff
@@ -206,6 +221,22 @@ def change_stream_quality(self, quality):
         self.stream.change_stream_quality(quality)
         self.player.play()
 
+    def open_stream_in_browser(self, event):
+        url = self.stream.url.lower()
+        url = ("http://www." if "www" not in url else "http://") + url
```

Comment:
Have a look at https://docs.python.org/3/library/urllib.parse.html for the URL parsing. It could help.